task/net/src/main.rs

```diff
@@ -85,6 +102,10 @@ fn main() -> ! {
     let mac = EthernetAddress::from_bytes(&FAKE_MAC);
 
     let ipv6_addr = link_local_iface_addr(mac);
+    ringbuf_entry!(Trace::Addr {
```

Comment:
You definitely don't need 16 elements in a ringbuffer that's only written once. If your goal is to discover the IP address of the Nucleo board, a global variable accessed by `readvar` would be much more space-efficient. (and also the IP address is fixed to `fe80::4:6ff:fe08:0a0c`)

Reply:
Frankly, I was playing and learning, and thought I might add other trace messages. But for checking it in for real, I'll nuke all this.

Thanks for the feedback!

Reply:
Because mac will only be known at runtime, what you're asking for is to use a mutable static, and Rust really would prefer I didn't do that (`error[E0133]: use of mutable static is unsafe and requires unsafe function or block`). I looked into the technique used by buf.rs, and I guess it would work, but it would be ugly for just this debug info. I also did it with unsafe, and that worked, but then the variable didn't end up visible with humility or objdump. Is there such a thing as "inlined statics"? I can't figure this out, so I nuked it for the time being.